zio-cli/shared/src/main/scala/zio/cli/ParserOptions.scala

```diff
@@ -5,9 +5,11 @@ package zio.cli
  * be parsed by ZIO CLI.
  *
  * @param caseSensitive     Whether or not to be case sensitive.
+ * @param autoCorrectLimit  Threshold for when to show auto correct suggestions
  */
 final case class ParserOptions(
-  caseSensitive: Boolean
+  caseSensitive: Boolean,
+  autoCorrectLimit: Int
```

Comment:
I think this is exactly the right place to put this!

However, I would suggest, it's time to rename `ParserOptions` to `CLIConfig`, because `autoCorrectLimit` is not really related to parsing anymore, just configuration data for the library.

Reply:
By the way, why `CLIConfig` and not `CliConfig`? Personally I find latter more readable 🙃

Reply:
I agree with you.

Then to be consistent, we should change `CLIApp` to `CliApp`.